plugins/kinet/README.md

```diff
@@ -2,12 +2,31 @@ KiNET Plugin
 ============
 
 This plugin creates a single device with multiple output ports. Each port
-represents a power supply. This plugin uses the V1 DMX-Out version of the
-KiNET protocol.
+represents a power supply. This plugin supports both the V1 DMX out and the
+V2 Port out modes of the KiNET protocol.
```

Comment:
They seem to style it as this:
```suggestion
represents a power supply. This plugin supports both the V1 DMX Out and the
V2 Port Out modes of the KiNET protocol.
```

Well actually they go for DMX OUT too, but it looks a bit odd to me.

Although as per the other discussion I think we should probably go more like:
```suggestion
represents a power supply. This plugin supports both the DMX Out (V1) and the
Port Out (V2) modes of the KiNET protocol.
```

Reply:
Fixed.